include/z64save.h

```diff
@@ -132,7 +134,7 @@ typedef struct {
     /* 0x0068 */ ItemEquips equips;
     /* 0x0074 */ Inventory inventory;
     /* 0x00D4 */ SavedSceneFlags sceneFlags[124];
-    /* 0x0E64 */ FaroresWindData fw;
+    /* 0x0E64 */ FaroresWindData fw_main;
```

Comment:
should be camelCase, so fwMain. Same goes for fwSecondary. Could also just make this an array if you want.